web/src/components/employer-rates/EmployerRatesQuestions.tsx

```diff
@@ -0,0 +1,98 @@
+import { ReactElement, useContext, useState } from "react";
+import { Heading } from "../njwds-extended/Heading";
+import { Content } from "@/components/Content";
+import { FormControl, FormControlLabel, Radio, RadioGroup } from "@mui/material";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { isUndefined } from "lodash";
+import { ProfileDataContext } from "@/contexts/profileDataContext";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+
+interface Props {
+  CMS_ONLY_enable_preview?: boolean;
+}
+
+export const EmployerRatesQuestions = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+  const { business } = useUserData();
+  const { setProfileData } = useContext(ProfileDataContext);
+
+  const employerAccessRadioValue = isUndefined(business?.profileData.employerAccessRegistration)
+    ? ""
+    : String(business?.profileData.employerAccessRegistration);
+
+  const [employerAccessRadio, setEmployerAccessRadio] = useState<string>(employerAccessRadioValue);
+
+  const employerAccessRadioValueIsTrue =
+    employerAccessRadio === "true" || props.CMS_ONLY_enable_preview;
+  const employerAccessRadioValueIsFalse =
+    employerAccessRadio === "false" || props.CMS_ONLY_enable_preview;
+
+  return (
+    <div className="bg-base-extra-light padding-205 margin-top-3 radius-lg">
+      <Heading level={2}>{Config.employerRates.employerAccessHeaderText}</Heading>
+
+      <Content>{Config.employerRates.employerAccessText}</Content>
+      <FormControl fullWidth>
+        <RadioGroup
+          name="employerAccess"
+          value={employerAccessRadio}
+          onChange={(event) => {
+            const value = event.target.value;
+            setEmployerAccessRadio(value);
+            setProfileData((prev) => ({
+              ...prev,
+              employerAccessRegistration: value === "true",
+            }));
+          }}
+        >
+          <FormControlLabel
+            style={{ alignItems: "center" }}
+            labelPlacement="end"
+            value="true"
+            control={<Radio color={"primary"} />}
+            label={Config.employerRates.employerAccessTrueText}
+          />
+
+          <FormControlLabel
+            style={{ alignItems: "center" }}
+            labelPlacement="end"
+            value="false"
+            control={<Radio color={"primary"} />}
+            label={Config.employerRates.employerAccessFalseText}
+          />
+        </RadioGroup>
+      </FormControl>
+
+      {employerAccessRadioValueIsTrue && (
+        <>
```

Comment:
Is this a good way to handle dynamic rendering - role="status" aria-live="polite"?